apps/CEventService.API/appsettings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=localhost,1433;Database=DesarrolloDB;User Id=SA;Password=P@pers123;Encrypt=True;TrustServerCertificate=True;"
```

Comment:
Change the credentials and delete them from the repository, exposing the access keys is not a good practice, create a .env and send it to the communication channel.

Reply:
Thx, I'll do that in my next commit.

Reply:
thanks